spec/features/view_pages_spec.rb

```diff
@@ -1,13 +1,13 @@
 require "rails_helper"
 
-RSpec.feature "view pages", type: :system do
+RSpec.feature "view pages", type: :feature do
```

Comment:
Do we need to specify the type for every feature spec in `spec/features/`. I added a support method for components to automatically add the type to specs found within `app/components`

Reply:
No you don't, this is already baked into rspec and will be set automatically if Capybara is required so I don't think we need to do/add anything here.